test_libs/pyspec/eth2spec/phase1/state_transition.py

```diff
@@ -0,0 +1,70 @@
+from . import spec
+
+from .spec import (
+    BeaconState,
+    BeaconBlock,
+    Slot,
+)
+
+from eth2spec.phase0.state_transition import (
+    process_operation_type,
+    process_operations as process_operations_phase0,
+)
+
+
+def process_operations(state: BeaconState, block: BeaconBlock) -> None:
+    process_operations_phase0(state, block)
+
+    process_operation_type(
+        state,
+        block.body.custody_key_reveals,
+        spec.MAX_CUSTODY_KEY_REVEALS,
+        spec.process_custody_key_reveal,
+    )
+
+    process_operation_type(
+        state,
+        block.body.early_derived_secret_reveals,
+        spec.MAX_EARLY_DERIVED_SECRET_REVEALS,
+        spec.process_early_derived_secret_reveal,
+    )
+
+
+def process_block(state: BeaconState,
+                  block: BeaconBlock,
+                  verify_state_root: bool=False) -> None:
+    spec.process_block_header(state, block)
+    spec.process_randao(state, block)
+    spec.process_eth1_data(state, block)
+
+    process_operations(state, block)
+    if verify_state_root:
+        spec.verify_block_state_root(state, block)
+
+
+def process_epoch_transition(state: BeaconState) -> None:
+    spec.process_justification_and_finalization(state)
+    spec.process_crosslinks(state)
+    # TODO: Eligible
```

Comment:
I think this still needs to be done -- the "eligible" function in 1_custody-game.md needs to be integrated into the process_penalties_and_exits function from the beacon chain.

Reply:
`process_penalties_and_exits` thing will be removed in #1116, so you can remove this TODO tag now!
```suggestion
```

Reply:
Yes, as I mentioned in a comment above, this whole file will be deleted. #1116 gets rid of `eligible`. It is on my list of TODOs